exporters/zipkin/src/zipkin_exporter_factory.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/zipkin/zipkin_exporter_factory.h"
+#include "opentelemetry/exporters/zipkin/zipkin_exporter.h"
+#include "opentelemetry/exporters/zipkin/zipkin_exporter_options.h"
+
+namespace http_client = opentelemetry::ext::http::client;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+
+std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> ZipkinExporterFactory::Create()
+{
+  ZipkinExporterOptions options;
+  return Create(options);
```

Comment:
Just like OTLP exporters, should we use `Create(ZipkinExporterOptions{})` to reduce copy here?

Reply:
Likewise, the Create() method with an option takes a `const &` parameter,
so there should be no copy.